Modules/Utils/Sources/PocketCastsUtils/Extensions/Double+Extensions.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+extension Double {
+
+    /// Checks whether the value is not NaN or Infinite
+    public var isValid: Bool {
```

Comment:
While I think `isNaN` configures a "not valid" value, I'm not so sure about infinite. Maybe we should call this `isNumeric`? Or `isANumber`?

Reply:
Ah! `isNumeric` is better, I wasn't sure what to call it. 🥳 

Done here: e9d735c18f3576ca2a5a6beb2a2635088d56a965